backend/vouch/trust_algo.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from vouch.models import Voucher
+from core.models import user
+
+# hyper parameter of algorithm - select them wisely
+a = 0.2  # define the minimum ratio of trust that is left to pre-trusted users
+eps = 0.00000001
+alpha = 0.1  # define how much voting weight goes to pre-trusted users for each user
+
+
+def normalize_trust_values(C, trust_status):
+    """
+    Normalize the trust values per user, and make users vouch for pre-trusted set.
+    If they vouch for other people, shift some of their ouching weight to others
+    """
+    trust_size = np.sum(np.array(trust_status) > 0, axis=0)  # size of the pre-trusted set
+    for i in range(len(C[0])):
+        vouching_size = np.sum(np.array(C[i]) > 0, axis=0)
+        for j in range(len(C[0])):
+            trust_indic = 1 if trust_status[j] > 0 else 0  # whether user j is pre-trusted
+            vouch_indic = 1 if C[i][j] > 0 else 0  # whether user j has been vouched by user i
+            C[i][j] = alpha / (alpha * trust_size + vouching_size) * trust_indic
+            C[i][j] += 1 / (alpha * trust_size + vouching_size) * vouch_indic
+    return C
```

Comment:
(minor)
Optimizing this function may not be very worthwhile with the current number of users, but most (if not all) this operation could probably be computed without python loop.
For example:
```py
trust_indic = (trust_status > 0).astype(int)
vouch_indic = (C > 0).astype(int)
```